use_cases/gretel.json

```diff
@@ -77,12 +77,12 @@
       "modelCategory": "synthetics",
       "defaultConfig": "config_templates/gretel/synthetics/natural-language.yml",
       "sampleDataset": {
-        "fileName": "taylor-swift-lyrics.csv",
+        "fileName": "taylor-swift-lyrics-up.csv",
```

Comment:
you'll want to generate a sample data preview for this new CSV file. There's a script in this repo to do so.

The change to this file is one we really should update in develop first and then cherry-pick to main, so that we don't have future issues when we promote things from develop -> main in our normal Console development workflow. Sorry for any inconvenience.

Reply:
@thesteady yep, Alex and I just discussed this step. @zredlined can you remove this file from the PR? There's some hard coded metadata about the sample dataset in gretel.json as well. I can take care of updating gretel.json and running the preview generation script in a separate PR in `develop`.

Reply:
oh, i see you updated the metadata as well.

Reply:
Generated the dataset preview and pushed it to this branch.